src/gui/symbology-ng/qgsrendererv2propertiesdialog.cpp

```diff
@@ -222,7 +222,6 @@ void QgsRendererV2PropertiesDialog::rendererChanged()
     {
       if ( mMapCanvas )
         mActiveWidget->setMapCanvas( mMapCanvas );
-      changeOrderBy( mActiveWidget->renderer()->orderBy(), mActiveWidget->renderer()->orderByEnabled() );
```

Comment:
If one creates a new 2.5D renderer and then switches to the standard one, is the order by imposed by the 2.5D renderer still in place?